be/src/Project/Services/Identity/Identity.Api/v1/Controllers/ApprovalsController.cs

```diff
@@ -133,6 +133,7 @@ public async Task<IActionResult> GetAsync(Guid? id)
         [HttpGet, MapToApiVersion("1.0")]
         [ProducesResponseType((int)HttpStatusCode.OK , Type = typeof(PagedResults<IEnumerable<ApprovalResponseModel>>))]
         [ProducesResponseType((int)HttpStatusCode.UnprocessableEntity)]
+        [AllowAnonymous]
```

Comment:
**question:** This AllowAnonymous tag is required?

Reply:
Yes, I need to get approvals for the SetPasswordForm.